core/src/stages/compute_ik.cpp

```diff
@@ -388,15 +406,17 @@ void ComputeIK::compute() {
 			solution.setComment(s.comment());
 			std::copy(frame_markers.begin(), frame_markers.end(), std::back_inserter(solution.markers()));
 
-			if (succeeded && i + 1 == ik_solutions.size())
+			if (ik_solutions[i].feasible)
```

Comment:
I guess this new condition just aims to be more obvious.
Do we still need the variable `succeeded`?

Reply:
It's [used below](https://github.com/ros-planning/moveit_task_constructor/pull/376/files#diff-eb4791e6fc52e29c2b9fb96a606928a1bf587989c5ebf29af186a50137ecc159R434).